### ISSUE TITLE
Suggestion: typo and tool

### ISSUE THREAD

**Issue by mboelen** [LewisVo/Awesome-Linux-Software#188]
Nice work and you were discovered by cron.weekly :)

Typo: in the description of this project is says 'softwares'. That should be software, as it is already a plural for multiple pieces.

Suggestion for your security section: Lynis, to perform a security scan of your Linux system - https://github.com/CISOfy/lynis

**Comment by LewisVo** [comment]
Hi there @mboelen, thank you for your commend 👍 I also subscribed to cron.weekly and have just seen my project got linked to on the latest issue ;). 

For the typo, I fixed that.  I am not a native speaker so sometimes words boggle my mind with their countability. 
For the Lynis one, I did a quick skim on the project and overall it looks promising. However, I am gonna need some time to check it out so I ll leave this open until then. ;) Thank you for your suggestion.
